whitelist.txt

```diff
@@ -355,4 +355,6 @@ Transaction2930
 
 downloader
 Downloader
-sublist
\ No newline at end of file
+sublist
+
+testnets
```

Comment:
```suggestion
```

We're the specification for mainnet. If we also happen to be useful to define testnets, so be it, but we should explicitly focus on mainnet.